pandas/core/groupby/generic.py

```diff
@@ -908,11 +908,15 @@ class DataFrameGroupBy(GroupBy):
     )
     @Appender(_shared_docs["aggregate"])
     def aggregate(self, func=None, *args, **kwargs):
-
         relabeling = func is None and is_multi_agg_with_relabel(**kwargs)
         if relabeling:
+            # OWO CHANGES
```

Comment:
We should make this into a new normalize_keyword_aggregation

Reply:
While we're at it, check if second field of tuple is a lambda, since native aggregation methods wouldn't work obviously